brave/browser/brave_browser_context.cc

```diff
@@ -532,21 +532,18 @@ void BraveBrowserContext::CreateProfilePrefs(
   bool async = false;
 
   if (IsOffTheRecord()) {
-    overlay_pref_names_.push_back("app_state");
-    overlay_pref_names_.push_back(extensions::pref_names::kPrefContentSettings);
-    overlay_pref_names_.push_back(prefs::kPartitionPerHostZoomLevels);
```

Comment:
Here's the actual important fix; NOT passing these to what because an inclusion list for incognito pref key names